R/app_server.R

```diff
@@ -16,6 +16,33 @@
     )
   })
 
+  # Display their name so they can log out if it's the wrong person. This should
+  # eventually move into shinyslack as an exported module.
+  slack_user_info <- shiny::reactive({
+    slack_info <- slackcalls::post_slack(
+      slack_method = "auth.test"
+    )
+
+    # Log the current user on shinyapps.
+    message(
```

Comment:
Does this give a pop up saying you are logged in as ? 
Not sure if this needed

Reply:
No, it just sends that message to the server, ie it logs it on shinyapps (or in your console if you run_app() locally). Useful for debugging. You can `usethis::pr_fetch(17)` to pull this down locally, and then build it and `run_app()` to see how it works. I also went ahead and deployed at shinyapps (r4ds.io/bookclubber?book_name=Mastering%20Shiny for example).